NEWS.md

```diff
@@ -2,21 +2,69 @@
 
 ## Polars R Package (development version)
 
+Updated rust-polars to 0.41.2 (#1147).
+
 ### Breaking changes
 
 - In `$n_chunks()`, the default value of `strategy` now is `"first"` (#1137).
--`$sample()` for Expr and DataFrame (#1136):
+
+- `$sample()` for Expr and DataFrame (#1136):
```

Comment:
My understanding is that if a blank line is inserted in a Markdown list, it may no longer be considered the same list. What is the intention of adding a blank line here?

Reply:
Ah I didn't know that. I just added that to make it easier to parse in raw mode but I'll change that if it breaks markdown